recipes/cppfront/all/test_package/conanfile.py

```diff
@@ -0,0 +1,46 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.layout import basic_layout
+from conans import tools as tools_legacy
+from conan.tools.build import cross_building
+import os
+
+from conan.errors import ConanException
+from io import StringIO
+import textwrap
+
+class TestPackageConan(ConanFile):
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _cppfront_input_path(self):
+        return os.path.join(self.build_folder, "pure2-hello.cpp2")
+
+    def build(self):
+        tools_legacy.save(self._cppfront_input_path, textwrap.dedent("""\
```

Comment:
This can be simplified. It is not necessary to build something to test that the package was correctly created. Running ``cppfront --version`` or equivalent could be more than enough.